barretenberg_static_lib/src/lib.rs

```diff
@@ -8,12 +8,11 @@ mod pippenger;
 pub mod scalar_mul;
 pub mod schnorr;
 use common::acvm::FieldElement;
-use std::convert::TryInto;
 
 pub struct Barretenberg;
 
 pub fn field_to_array(f: &FieldElement) -> [u8; 32] {
-    let v = f.to_bytes();
+    let v = f.to_be_bytes();
```

Comment:
Once again, just hoping this does not break anything else. Can we make sure this backend upgrade is also tested using all the tests under `nargo/tests/test_data`?

Reply:
Replied above -- in Noir there is migrate to 0.3 PR open that runs the tests